frontend/src/components/program/picasso/PicassoEntry.vue

```diff
@@ -379,6 +380,8 @@ export default {
 }
 .e-picasso-entry--editable:hover,
 .e-picasso-entry--temporary {
+  cursor: default;
```

Comment:
Why was this added?

Reply:
For the temporary placeholder, I don't want to show the grab pointer (which is for move operation of existing events). This change makes the switching back and forth of different cursor animations a bit less flaky. But still remains a cosmetic issue to fix.